spec/rubocop/cop/layout/space_before_block_braces_spec.rb

```diff
@@ -63,11 +63,6 @@
       expect(cop.config_to_allow_offenses).to eq('Enabled' => false)
     end
 
-    it 'auto-corrects unwanted space' do
-      new_source = autocorrect_source('each {}')
-      expect(new_source).to eq('each{}')
-    end
```

Comment:
This test was removed because it was reliant on the default setting. It also was essentially a dup of [this test](https://github.com/bbatsov/rubocop/blob/master/spec/rubocop/cop/layout/space_before_block_braces_spec.rb#L96), where the config is explicitly set.